docs/legacy/porting1214.md

```diff
@@ -1,6 +0,0 @@
-Porting from Minecraft 1.12 to Minecraft 1.13/1.14
-==================================================
-
-Due to major rewrites to Forge happening during the lifetime of 1.13 it never saw much usage and instead most mods ported directly to 1.14 skipping over 1.13. Therefore this guide will focus on porting from 1.12 directly to 1.14.
-
-_Full documentation on porting from 1.12 to 1.14 is still work in progress, for now we suggest having a look at [a primer written by williewillus](https://gist.github.com/williewillus/353c872bcf1a6ace9921189f6100d09a)!_
```

Comment:
I don't see the point in removing this without replacement. These guides are still useful to someone coming from versions before 1.15.